app/controllers/admin/api/base_controller.rb

```diff
@@ -129,11 +129,15 @@ def authorize_service_plans!
   ##~ @parameter_service_id_by_id = {:name => "id", :description => "ID of the service.", :dataType => "int", :required => true, :paramType => "path", :threescale_name => "service_ids"}
   ##~ @parameter_service_id_by_id_name = {:name => "service_id", :description => "ID of the service.", :dataType => "int", :required => true, :paramType => "path", :threescale_name => "service_ids"}
 
-  ## BackendAPIs
+  ## Backend APIs
 
   ##~ @parameter_backend_api_id_by_id = { :name => "id", :description => "ID of the Backend.", :dataType => "int", :required => true, :paramType => "path" }
   ##~ @parameter_backend_api_id_by_id_name = { :name => "backend_api_id", :description => "ID of the Backend.", :dataType => "int", :required => true, :paramType => "path" }
 
+  ## Backend API Configs
```

Comment:
@thomasmaas So we really need to choose: `Config` or `Usage` as this is public documentation

Reply:
@hallelujah it is usage and we already decided it as it is explained in the document linked in the description, but this comment is not part of the public documentation, this is just the base controller where we declare each 'id' param and and we organize it inside models like I am doing here but this won't be public. Look at more examples:
https://github.com/3scale/porta/blob/12c1b2d39cb540fcd614db499c896b0d18db9ccc/app/controllers/admin/api/base_controller.rb#L97-L125